### ISSUE TITLE
Error: Cannot find module '../dist'

### ISSUE THREAD

**Issue by pine** [lukeed/taskr#138]
I've just update to fly v0.9.0, but it failed.

```
module.js:327
    throw err;
    ^

Error: Cannot find module '../dist'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/pine/project/fly-ava/node_modules/fly/bin/_index:3:1)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
```

I think that `_index` file can't load `"../dist"`directory because `dist` directory is not included in `package.json`

https://github.com/bucaran/fly/blob/a2a7467027f4076acb3cf018ebf90af04ecf4b73/bin/_index
https://github.com/bucaran/fly/blob/a2a7467027f4076acb3cf018ebf90af04ecf4b73/package.json#L6-L9

Thank you.

**Comment by lukeed** [comment]
Whoops! Sorry about that. Fixing it now

**Comment by lukeed** [comment]
Fixed in 9.0.1 

Thanks for reporting, I would have discovered it tomorrow anyway :)

**Comment by pine** [comment]
:+1:
